<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>control_app</class>
 <widget class="QWidget" name="control_app">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="system_mode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>401</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Alignment mode settings&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="Setup">
    <attribute name="title">
     <string>Setup</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Preview">
    <attribute name="title">
     <string>Preview</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Acquire">
    <attribute name="title">
     <string>Acquire</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Exit">
    <attribute name="title">
     <string>Exit</string>
    </attribute>
    <widget class="QPushButton" name="exit_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Push to exit</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="system_status">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>331</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>System status</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Laser</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>On/Off</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Power</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wavelength</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Motor position</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="onoff">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="power">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="wavelength">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="mode">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="mtpos">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Positioning</string>
    </property>
    <widget class="QLineEdit" name="zpos">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="xpos">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ypos">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
